### ISSUE TITLE
fast_dev_run -> unit_test

### ISSUE THREAD

**Issue by williamFalcon** [PyTorchLightning/pytorch-lightning#1081]
Anyone want to make this change?

- rename fast_dev_run -> unit_test
- add checking the test set as well (currently only checks val, train).

**Comment by Gokkulnath** [comment]
Hi @williamFalcon , I would like to take up this opportunity. Can you guide me with the changes required for addition of test set check?

**Comment by Borda** [comment]
closing since we find out an appropriate name...
